templates/repo/issue/view_content/add_reaction.tmpl

```diff
@@ -1,11 +1,9 @@
-{{if .ctxData.IsSigned}}
+{{if ctx.RootData.IsSigned}}
 <div class="item action ui dropdown jump pointing top right select-reaction" data-action-url="{{.ActionURL}}">
-	<a class="add-reaction muted">
-		{{svg "octicon-smiley"}}
-	</a>
-	<div class="menu reactions-menu">
+	<a class="muted">{{svg "octicon-smiley"}}</a>
+	<div class="menu">
 		{{range $value := AllowedReactions}}
-			<a class="item reaction" data-tooltip-content="{{$value}}" aria-label="{{$value}}" data-reaction-content="{{$value}}">{{ReactionToEmoji $value}}</a>
+		<a class="item comment-reaction-button" data-tooltip-content="{{$value}}" aria-label="{{$value}}" data-reaction-content="{{$value}}"><span class="emoji">{{ReactionToEmoji $value}}</span></a>
```

Comment:
I think we can make this a `<button>` while we are at it.

Reply:
Button doesn't work well in dropdown, and it needs a lot of "fine-tunes" like border/hover.

I would prefer to keep it, or just add "role" for it.

Reply:
Well, no need to add anything.

Because, we have our "dropdown.js", so "<a>" is always preferred for a dropdown item.

Reply:
Alright. I still would want to use `<button>` in future dropdowns, but it's something for later.

Reply:
Maybe we can add `role=button` at least in the reactions row.

Reply:
No, they already have "role=menuitem" automatically, by our "dropdown.js". It just works well.

Reply:
I meant the ones below the content, but I see they already have `button` role.